interactive_assign_nodes.py

```diff
@@ -7,21 +7,22 @@ def main():
     user_example_file = "/Users/lucas/OneDrive - The University of Colorado Denver/Grants/Cartoomics-Grant/Test_Data/DS/DS_example_input.csv"
 
     # # This should be turned into a config file I think, or CLI with a directory input and we should also add a logger
-    # triples_list_file = '/Users/brooksantangelo/Documents/HunterLab/GutMGene_PKL/Output/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_Identifiers_withGutMGene_withMicrobes.txt'
+    triples_list_file = '/Users/lucas/OneDrive - The University of Colorado Denver/knowledge-graphs/pkl/instance_relationsOnly_OWLNETS/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_Identifiers.txt'
 
-    # labels_file = '/Users/brooksantangelo/Documents/HunterLab/GutMGene_PKL/Output/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_NodeLabels_NewEntities.txt'
+    labels_file = '/Users/lucas/OneDrive - The University of Colorado Denver/knowledge-graphs/pkl/instance_relationsOnly_OWLNETS/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_NodeLabels.txt'
 
-    # identifiers_file = '/Users/brooksantangelo/Documents/HunterLab/GutMGene_PKL/Output/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_Integer_Identifier_Map_withGutMGene_withMicrobes.json'
+    identifiers_file = '/Users/lucas/OneDrive - The University of Colorado Denver/knowledge-graphs/pkl/instance_relationsOnly_OWLNETS/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_Integer_Identifier_Map.json'
 
-    # triples_integers_file = '/Users/brooksantangelo/Documents/HunterLab/GutMGene_PKL/Output/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_node2vecInput_cleaned_withGutMGene_withMicrobes.txt'
+    triples_integers_file = '/Users/lucas/OneDrive - The University of Colorado Denver/knowledge-graphs/pkl/instance_relationsOnly_OWLNETS/PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_Integers.txt'
 
     # output_dir = '/Users/brooksantangelo/Documents/HunterLab/Cartoomics/Scripts/Test'
+    
+    g = create_pkl_graph(triples_list_file,labels_file)
 
     u = read_user_input(user_example_file)
-    n = map_nodes(u)
-    print(n)
-    
-    #g = create_pkl_graph(triples_list_file,labels_file,identifiers_file,triples_integers_file)
+    n = unique_nodes(u)
+    search_nodes(n,g)
```

Comment:
does this need to return to a list? If we want to input this to the prioritize paths function (or other path search) then it could be in the form of a df with Source/Target columns

Reply:
otherwise this branch looks good, I think we just have to figure out how to connect them all

Reply:
this doesn't need to return a list so I can map it to the dataframe like you suggested. I'll update with the next pull request.